.github/workflows/reusable-e2e.yml

```diff
@@ -229,9 +262,14 @@ jobs:
           aws configure set aws_session_token $AWS_SESSION_TOKEN && \
           aws configure set default.region $AWS_REGION
 
-      - name: Pull down Storage AWS environments
+      - name: Pull down Gen1 Storage AWS environments
+        if: ${{ matrix.version == 'gen1' }}
         run: yarn environments storage pull
 
+      - name: Pull down Gen2 Storage AWS environments
+        if: ${{ matrix.version == 'gen2' }}
+        run: yarn environments storage pull -v gen2
```

Comment:
Can we just iterate over the values of `matrix.version` and run these by passing the version to the pull script? Something like:

```
      - name: Pull down ${{ matrix.version }} Storage AWS environments
        if: ${{ matrix.version == 'gen2' }}
        run: yarn environments storage pull -v ${{ matrix.version }}
```

Reply:
Good idea. That way we do not need the if conditional as well. I will update and give it a try 👍